app/_assets/stylesheets/base.less

```diff
@@ -359,3 +359,28 @@ nav {
   height: 0;
   width: 0;
 }
+
+.hiring-badge {
+  display: none;
+}
+
+.hiring-badge span {
+  font-size: 13px;
+  text-transform:uppercase;
+  color:#1456cb;
+  background:#ecf6ff;
+  border-radius:5px;
+  display:inline-block;
+  padding:5px 15px;
```

Comment:
```suggestion
  padding:5px 15px;
  vertical-align: text-top;
```
It's not quite aligned at the moment.
![Screen Shot 2022-03-22 at 3 37 45 PM](https://user-images.githubusercontent.com/54370747/159587889-d752f3b1-1ba5-42bc-bd28-ae5847403531.png)